src/parser/expression/parsed_function_expression.cpp

```diff
@@ -14,12 +14,17 @@ std::unique_ptr<ParsedFunctionExpression> ParsedFunctionExpression::deserialize(
     deserializer.deserializeValue(isDistinct);
     std::string functionName;
     deserializer.deserializeValue(functionName);
-    return std::make_unique<ParsedFunctionExpression>(std::move(functionName), isDistinct);
+    std::vector<std::string> optionalArguments;
```

Comment:
I assume this relates to a bug where STRUCT is used as default value. Though I don't see a corresponding test.

Reply:
The new CI jobs have covered it with existing test cases.